### ISSUE TITLE
After building from source running fails due to no config

### ISSUE THREAD

**Issue by Jibodeah** [daid/EmptyEpsilon#1968]
I'm trying to run the game on Fedora 37 (the package in RPMFusion is sadly out of date, AND suffers from the same problem I'm about to describe). I've built the game from source (I've tried both from the master branch and the `EE-2022-10-28` tag), and it builds successfully. But when running the game I get a popup saying "Failed to load gui theme, resources missing?". Log follows

```
INFO: Starting...
INFO: Using /home/jibodeah/.emptyepsilon as configuration path
ERROR: Failed to open gui/default.theme.txt for tree loading
ERROR: Failed to load default theme, exiting.
```

...and sure enough, there is no `~/.emptyepsilon` directory.

Have I missed something? I feel like it should auto generate this directory with defaults if it is missing, or put the directory there as part of the build procedure.

**Comment by aBlueShadow** [comment]
First I would check if there is something wrong with your download, here is the theme file https://github.com/daid/EmptyEpsilon/blob/master/resources/gui/default.theme.txt, can you find it in your source directory?
Yes, the config directory should be created , but it might be that the error happens before that step.

**Comment by Jibodeah** [comment]
Yes that file exists. I cloned the repo with git so I don't think missing files is the issue.

Do you know at what stage in the build process the config directory is supposed to be created? I've just tried building again but no log messages in either the cmake or ninja step seem pertinent.

**Comment by daid** [comment]
This has nothing to do with config. You are just running it from the wrong working directory and thus cannot find any resource files.

**Comment by Jibodeah** [comment]
Ah

Well in that case could that be mentioned on the wiki (or in the log) somewhere? I generally don't expect things to care about what working directory they're run from.
